apps/events/templatetags/shuffle.py

```diff
@@ -6,5 +6,5 @@
 @register.filter
 def shuffle(arg):
     li = list(arg)[:]
```

Comment:
`arg[:]` is a trick used to create a shallow copy of the list.
Though the same is achieved by using `list()`, so in this case it is redundant. 

Sidenote: `arg[0:-1]` is not the same as `arg[0:len(arg)]`. The first one will remove the last element since slices are not inclusive.

Reply:
Good point, forgot that they are exclusive, which means it's the same as `[...][0:]`, which in turn is the same as `[...][:]`. Though if they accomplish the same, I think `list(...)` should be used, as it is more explicit, and easier to understand.